metabase/metabase
# Add `TableInfoPopover` and add it to Table cards on the `/browse` route

#18738

This PR adds a "TableInfoPopover" to table cards found on the `/browse` route, navigated to via the "Browse Data" button in the header.

**Testing**
1. Navigate to the `/browse` route and select a database
2. Hover over table cards to see the popover
3. On second hover, you shouldn't see a loading state for field/fk metadata
4. Click on "connected tables" links to be navigated to a simple question view of the specified table

![Screen Shot 2021-12-13 at 1 37 01 PM](https://user-images.githubusercontent.com/13057258/145892784-c58c26c1-eb39-4892-9aef-e7a9eb19cdc4.png)


https://user-images.githubusercontent.com/13057258/146834300-28b23441-6aa2-488c-a060-0f3f5f7d444d.mov
